app/forms/vehicle_details_form.rb

```diff
@@ -26,4 +27,6 @@ class VehicleDetailsForm
 
   attribute :vehicle_in_regular_use, :boolean
   validates :vehicle_in_regular_use, inclusion: { in: [true, false], allow_nil: false }
+
+  attribute :vehicle_in_dispute, :boolean, default: false
```

Comment:
Just worked out how to do this using govuk_check_box. It looks like there is a subtle bug that means that the 'magic' rails value doesn't show up by default, but passing a zero as the 3rd argument to govuk_check_box makes everything come together (rather than the default of false)